pkg/api/validation/schema.go

```diff
@@ -62,27 +62,36 @@ type NullSchema struct{}
 func (NullSchema) ValidateBytes(data []byte) error { return nil }
 
 type SwaggerSchema struct {
-	api swagger.ApiDeclaration
+	api     swagger.ApiDeclaration
+	factory Schema // For delegating to  other api groups
```

Comment:
I can change it to delegate.  It is named factory because that is the name of the Schema type that it actually delegates to.  WDYT?